src/api/spec/controllers/source_controller_spec.rb

```diff
@@ -207,4 +207,34 @@
       it { expect(response).to have_http_status(:ok) }
     end
   end
+
+  describe 'POST #linktobranch' do
```

Comment:
You are testing `if`...

Reply:
Now, yes. Initially I didn't know... But you are right, I can remove that test.